src/assets/styles/main.scss

```diff
@@ -1,5 +1,5 @@
 @import 'variables';
-@import 'fontawesome-free-5.10.2-web/scss/fontawesome';
-@import 'fontawesome-free-5.10.2-web/scss/solid';
-@import 'fontawesome-free-5.10.2-web/scss/regular';
-@import 'fontawesome-free-5.10.2-web/scss/brands';
+@import '../../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome.scss';
```

Comment:
@Yuriy-27 there is good practice to get all lib scss imports in the separate scss file called as `vendor.scss` or some another name. And import that one file into main.scss